docs/_src/tutorials/tutorials/11.md

```diff
@@ -405,7 +405,7 @@ pipelines:    # multiple Pipelines can be defined using the components from abov
 
 To load, simply call:
 ``` python
-pipeline.load_from_yaml(Path("sample.yaml"))
+yaml_pipeline = Pipeline.pipeline.load_from_yaml(Path("sample.yaml"))
```

Comment:
I think `Pipeline` has no `pipeline` attribute

```suggestion
yaml_pipeline = Pipeline.load_from_yaml(Path("sample.yaml"))
```

Reply:
Right, that was by mistake, fixed, thanks!